content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -72,6 +72,35 @@ violated from the PodSecurity enforcement.
 See [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
 for more information.
 
+## apiserver.latency.k8s.io/transform-response-object
+
+Example: `apiserver.latency.k8s.io/transform-response-object: "1.386µs"`
+
+This annotations shows the latency incurred in transforming the response object(s)
```

Comment:
This description doesn't tell me a lot about when this annotation gets added. What is an object here? What does "user's desired form" mean to me, a cluster operator? When does this latency come into play? 

Also, is the value for this annotation actually showing the unit alongside the number? We should say what the unit is in the text as well. 

Are there any docs that describe these operations in more detail, which we can link to?